### PR TITLE
Update Jabel for PKware usage

### PR DESCRIPTION
**Pull request by mbayerPK**
Change Jabel project group from  "com.github.bsideup.jabel" to "com.pkware.jabel"

Update settings.gradle and all project build.gradle files to kotlin and use .kts extensions

Remove original Jabel publishing workflow and add PKware third party publishing workflow

Add project versioning in addition to tags

Add gradle.properties file

Update README.md for kotlin and add PKware third party release/publish instructions

Set initial version to 1.0.2-PKWARE-SNAPSHOT

### CODE REVIEW

**Review by MatthewFoster624 — APPROVED** [review]

File: gradle.properties [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Default value: -Xmx10248m -XX:MaxPermSize=256m
+# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError
+
+org.gradle.parallel=true
+org.gradle.daemon=true
+org.gradle.caching=true
+org.gradle.jvmargs=-Dfile.encoding=UTF-8
+kotlin.code.style=official
+
+jabelVersion=1.0.2-PKWARE-SNAPSHOT
```
Comment (MariusVolkhart) [inline gradle.properties]: Let's go with `1.0.1-1` - it's version 1.0.1 of the upstream code, and it's our first release of that "code snapshot"

File: jabel-javac-plugin/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+                scm {
+                    connection.set("scm:git:git://github.com/pkware/jabel.git")
+                    developerConnection.set("scm:git:ssh://github.com/pkware/jabel.git")
+                    url.set("https://github.com/pkware/jabel")
+                }
+
+                licenses {
+                    license {
+                        name.set("Apache License, Version 2.0")
```
Comment (MariusVolkhart) [inline jabel-javac-plugin/build.gradle.kts:58]: Double check that this license is the same as the original repository

File: jabel-javac-plugin/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+            from(components["java"])
+            pom {
+                name.set(pomName)
+                packaging = pomPackaging
+                description.set(pomDescription)
+                url.set("https://github.com/pkware/jabel")
+                setPkwareOrganization()
+
+                developers {
+                    developer {
```
Comment (MariusVolkhart) [inline jabel-javac-plugin/build.gradle.kts:43]: If the original library had a developer that wasn't us, we should keep that. Again, we're not really doing anything here except publishing

**Review by MariusVolkhart — APPROVED** [review]

File: jabel-javac-plugin/build.gradle.kts (hunk 2 above)
Comment (mbayerPK) [reply jabel-javac-plugin/build.gradle.kts:58]: Aye it is, see lines 9-14 in the deleted gradle/publishing.gradle file

**Pull request merged**